samples/client/petstore/php/SwaggerClient-php/phpunit.xml.dist

```diff
@@ -14,8 +14,8 @@
 
     <filter>
         <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">./lib/Api</directory>
-            <directory suffix=".php">./lib/Model</directory>
+            <directory suffix=".php">./lib\Api</directory>
```

Comment:
@wing328 PHP doesn't care about that, it will auto-adjust the separator to work on Windows.